pinot-core/src/main/java/org/apache/pinot/core/operator/ProjectionOperator.java

```diff
@@ -68,6 +72,36 @@ public String getOperatorName() {
     return OPERATOR_NAME;
   }
 
+  @Override
+  public String getExplainPlanName() {
+    return EXPLAIN_NAME;
+  }
+
+  @Override
+  public List<Operator> getChildOperators() {
+    return Collections.singletonList(_docIdSetOperator);
+  }
+
+  @Override
+  public String toExplainString() {
+    StringBuilder stringBuilder = new StringBuilder(getExplainPlanName()).append('(');
+    if (_dataSourceMap.keySet().isEmpty()) {
+      // count aggregation function has empty input expressions
+      stringBuilder.append("ALL");
```

Comment:
I don't follow this. `PROJECT ALL` makes sense when we are projecting / selecting all columns. However, this comment is implying that we will do `PROJECT ALL` for count aggregation function which doesn't sound right

Reply:
The comment appears to be incorrect. I don't see this block of code being invoked when `count(*)` is used. There is a test case for `count(*)` query.